app/models/speakers.rb

```diff
@@ -0,0 +1,2 @@
+class Speakers < Metric
```

Comment:
This doesn't seem right. Don't we have affordances for speakers in the transcripts model? What is this for?